eras/dijkstra/test-suite/cardano-ledger-dijkstra-test.cabal

```diff
@@ -0,0 +1,49 @@
+cabal-version: 3.0
+name: cardano-ledger-dijkstra-test
+version: 1.3.1.1
```

Comment:
This version number was taken from `cardano-ledger-conway-test` and does not make much sense, but this package is going to be removed/restructured anyway.

Reply:
No longer relevant, since I removed this package altogether.